frontend/src/components/filters/FiltersInfoTS.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { FILTER_TYPE } from "../../constants";
+
+
+const FiltersInfos = ({
+    filters,
+}) => {
+    const appliedFilters = Object.keys(filters).filter((key: any) => filters[key]?.value)
+    const getValue = (key: any) => {
+        const filterValue = filters[key].value
+        const description: any = filters[key].description
+        let valueJSX: any;
+        const descriptionJSX: any = (
+            <>
+                {
+                    <strong>{description.label}: </strong>
+                }
+            </>
+        );
+        const valuesArray = [].concat(filterValue)
+        let labels: any = [];
+        let value = ""
+        switch (description.type) {
+            case FILTER_TYPE.SELECT:
+                labels = valuesArray.map((val: any) => {
+                    if (description.options) {
+                        const option = description.options.find((option: { value: any }) => option.value === val)
+                        valueJSX = option.label
```

Comment:
Since this is an array mapping function, it should be returning a value, not setting a variable. This is why the filter strings for Sample Kind and Depleted aren't showing up. The `labels` variable should end up holding an array of value strings, but right now it is just empty.